From 842fd986eb561d0e5fe63d8a11b7ab72d3777c3c Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Sun, 31 Jul 2022 19:51:28 +0200
Subject: [PATCH 1/4] Imake: Fix build with binutils 2.36

Remove -l flag from Ar*Cmd; this flag was traditionally unused/ignored by
both GNU and BSD variants.

In binutils 2.36, GNU ar modified it's behavior to use it to specify
library dependencies.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 Xvnc/config/cf/Imake.tmpl | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/Xvnc/config/cf/Imake.tmpl b/Xvnc/config/cf/Imake.tmpl
index f04b4b1435c9..cfadbae79923 100644
--- a/Xvnc/config/cf/Imake.tmpl
+++ b/Xvnc/config/cf/Imake.tmpl
@@ -595,25 +595,13 @@ TCLIBDIR = TclLibDir
 #define ArCmdBase ar
 #endif
 #ifndef ArCmd
-#if HasLargeTmp || SystemV4
 #define ArCmd ArCmdBase cq
-#else
-#define ArCmd ArCmdBase clq
-#endif
 #endif
 #ifndef ArAddCmd
-#if HasLargeTmp || SystemV4
 #define ArAddCmd ArCmdBase ru
-#else
-#define ArAddCmd ArCmdBase rul
-#endif
 #endif
 #ifndef ArExtCmd
-#if HasLargeTmp || SystemV4
 #define ArExtCmd ArCmdBase x
-#else
-#define ArExtCmd ArCmdBase xl
-#endif
 #endif
 #ifndef BootstrapCFlags
 #define BootstrapCFlags /**/
-- 
2.37.1

